### ISSUE TITLE
Importing JSON with apostrophes in obsidian

### ISSUE THREAD

**Issue by Myliak** [farling42/obsidian-import-json#53]
Hello,

I am trying to import beast from 5e.tools json. The description of one of the abilities has apostrophes and when imported into obsidian, apostrophes are replaced with &#x27; which is unicode representation of '. Json in question is below

```
{
	"name": "Regeneration",
	"description": "The troll regains 10 hit points at the start of its turn. If the troll takes acid or fire damage, this trait doesn't function at the start of the troll's next turn. The troll dies only if it starts its turn with 0 hit points and doesn't regenerate."
}
```

And this is the imported text in Obsidian:

Regeneration: The troll regains 10 hit points at the start of its turn. If the troll takes acid or fire damage, this trait doesn&#x27;t function at the start of the troll&#x27;s next turn. The troll dies only if it starts its turn with 0 hit points and doesn&#x27;t regenerate.

Is there any way to bypass this behavior? I was trying to reproduce it with javascript, but I think it's problem with communication between obsidian and plugin.

Thank you for any replies.

**Comment by farling42** [comment]
I can't see a don't difference in your report.

Is it all three occurrences of the apostrophe which are being replaced?

**Comment by Myliak** [comment]
Github formatted it automatically, didn't notice, sorry :D

`Regeneration The troll regains 10 hit points at the start of its turn. If the troll takes acid or fire damage, this trait doesn&#x27;t function at the start of the troll&#x27;s next turn. The troll dies only if it starts its turn with 0 hit points and doesn&#x27;t regenerate.`

**Comment by farling42** [comment]
Oh, the HTML form of the character. (Unicode world be changing it's binary value to something like 2019)

**Comment by farling42** [comment]
This issue should be fixed in version 0.31.2
